conda/recipes/libcuspatial/conda_build_config.yaml

```
@@ -11,7 +11,7 @@ cmake_version:
   - ">=3.23.1,!=3.25.0"
 
 gtest_version:
-  - "1.10.0"
+  - "1.13.0"
```

the conda environments were changed to `>=1.13.` but it looks like this is pinned strictly to `1.13.0`.

was this intentional? shouldn't all of these values be consistent?

---

Yes. This should be updated.
```suggestion
  - ">=1.13.0"
```